ui/server/src/test/scala/pl/touk/nussknacker/ui/process/marshall/UiProcessMarshallerSpec.scala

```diff
@@ -12,7 +12,7 @@ class UiProcessMarshallerSpec extends FlatSpec with Matchers {
   val processWithAdditionalFields =
     s"""
        |{
-       |    "metaData" : { "id": "custom", "typeSpecificData": {"type": "StreamMetaData", "parallelism" : 2}, "additionalFields": { "description": "$someProcessDescription"} },
+       |    "metaData" : { "id": "custom", "typeSpecificData": {"type": "StreamMetaData", "parallelism" : 2}, "additionalFields": {"description": "$someProcessDescription", "groups": [], "properties": {}}},
```

Comment:
I think it should be left as it was before. Am I right?

Reply:
You are right. One test depends on previous structure. But at the same time, this is the marshalling logic I would expect. Thus, I've changed data for `marshall and unmarshall process` test.